test/modules/vass/use-depth.good

```diff
@@ -1,2 +1,6 @@
 use-depth.chpl:21: error: symbol x is multiply defined
 use-depth.chpl:37: note: also defined here
+use-depth.chpl:22: error: symbol y is multiply defined
+use-depth.chpl:28: note: also defined here
+use-depth.chpl:23: error: symbol z is multiply defined
+use-depth.chpl:33: note: also defined here
```

Comment:
Michael, this is obviously fine-tuning as well as being a pre-existing condition that you've inherited, but as I read through these errors, I'm realizing that while they helpfully point to where the conflicting symbols are declared, they don't point to the scope or use where the symbols are brought into conflict with one another.  I'm imagining, for example, a complaint saying `LibraryOne.foo` conflicts with `LibraryTwo.foo` without giving me any indication where those libraries are used by me or some other library could be confusing to know how to track down in a real program.  In general, I suppose someone might want this to be a complete list of the use chains that lead from the problematic scope to the symbols; but even just indicating that the conflicts were due to the uses of M1 and M2 on lines 7-8 would be a lot better than the status quo (because from there, I can manually walk the use chains if need be; but there's no real way to walk from the variable declaration points to the place where the conflict occurs if their modules are used in multiple places within the program without exhaustively checking everything.

If the disclaimer I kicked that off with didn't make it sufficiently clear, I'd have no problem forking this off into its own issue / PR rather than trying to solve it here.

Reply:
Sure, and I agree that would be good to improve. Since I am expecting that we are not very far from using the dyno scope resolver I created this issue https://github.com/Cray/chapel-private/issues/3166 to track the task of updating the dyno scope resolver to improve this error.